x/version/chains.go

```diff
@@ -0,0 +1,25 @@
+package version
+
+const (
+	MochaChainID          = "mocha"
+	ArabicaChainID        = "arabica-4"
+	BlockspaceraceChainID = "blockspacerace-0"
+	MainnetChainID        = "celestia-1"
+)
+
+// StandardChainVersions returns a map of chain IDs to their respective
+// ChainVersionConfig. Each hardfork should be added to this map.
+func StandardChainVersions() map[string]ChainVersionConfig {
+	version0Only := NewChainVersionConfig(map[uint64]int64{
+		0: 0,
+	})
+	mainnetVersions := NewChainVersionConfig(map[uint64]int64{
+		1: 0,
```

Comment:
[question] why doesn't this look something like:

```
0: 0,
1: 1000, // replace 1000 with the first upgrade block height
```

In other words, why start mainnet at `appVersion = 1`?

Reply:
the app version can be arbitrary, so I'm assuming that it will match the versions in github as much as possible. Hence starting mainnet at version 1